src/main/java/pl/edu/agh/firecell/renderer/RenderMode.java

```diff
@@ -0,0 +1,5 @@
+package pl.edu.agh.firecell.renderer;
+
+public enum RenderMode {
+    Normal, Air_Temperature, Solid_Temperature
```

Comment:
```suggestion
    NORMAL, 
    AIR_TEMPERATURE,
    SOLID_TEMPERATURE
```
That's convetion in Java